pytorch_lightning/accelerators/accelerator.py

```diff
@@ -385,7 +397,9 @@ def broadcast(self, obj: object, src: int = 0) -> object:
         """
         return self.training_type_plugin.broadcast(obj, src)
 
-    def all_gather(self, tensor: Union[torch.Tensor], group: Optional[Any] = None, sync_grads: bool = False):
+    def all_gather(
+        self, tensor: Union[torch.Tensor], group: Optional[Any] = None, sync_grads: bool = False
```

Comment:
Could this be simplified?:
```suggestion
        self, tensor: torch.Tensor, group: Any = None, sync_grads: bool = False
```

Reply:
For the first part yes, but I'd still have the Optional here, since this IMO helps with readibility, since it means it could be an object but also None